cmd/manager/main.go

```diff
@@ -178,6 +178,11 @@ func Command() *cobra.Command {
 		container.DefaultContainerRegistry,
 		"Container registry to use when downloading Elastic Stack container images",
 	)
+	cmd.Flags().String(
+		operator.ContainerSuffixFlag,
+		"",
+		fmt.Sprintf("Suffix to be appended to container images by default. Cannot be combined with %s", operator.UBIOnlyFlag),
```

Comment:
nit: maybe do the same for the UBI flag?

```golang
	cmd.Flags().Bool(
		operator.UBIOnlyFlag,
		false,
		fmt.Sprintf("Use only UBI container images to deploy Elastic Stack applications. UBI images are only available from 7.10.0 onward. Cannot be combined with %s", operator.ContainerSuffixFlag),
	)
```

And do a similar change in the doc?